testdata/repros/Issue3653.t.sol

```diff
@@ -11,13 +11,13 @@ contract Issue3653Test is DSTest {
     Token token;
 
     constructor() {
-        fork = vm.createSelectFork("rpcAlias", 10);
+        fork = vm.createSelectFork("rpcAlias", 1000000);
```

Comment:
block 10 had gas limit of 5000 which is not enough do deploy a contract in isolated mode